examples/chaincode/go/asset_management_with_roles/asset_management_with_roles.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package main
```

Comment:
@binhn I have pushed this comment changes. The link was removed temporarily we could add this again once the content will be fixed and consistent with the implementation. 

I have removed the duplicated comment and added a description of the sample functionality.